src/main/java/com/stripe/model/Account.java

```diff
@@ -1,607 +1,735 @@
-      throws StripeException {
-    return request(RequestMethod.DELETE, instanceUrl(Account.class, this.id), params,
-        Account.class, options);
+  public Account update(Map<String, Object> params, RequestOptions options) throws StripeException {
+    String url =
+        String.format("%s%s", Stripe.getApiBase(), String.format("/v1/accounts/%s", this.getId()));
+    return request(ApiResource.RequestMethod.POST, url, params, Account.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="list">
   /**
-   * List all connected accounts.
+   * Returns a list of accounts connected to your platform via <a href="/docs/connect">Connect</a>.
+   * If you’re not a platform, the list is empty.
    */
   public static AccountCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+    return list(params, (RequestOptions) null);
   }
 
   /**
-   * List all connected accounts.
+   * Returns a list of accounts connected to your platform via <a href="/docs/connect">Connect</a>.
+   * If you’re not a platform, the list is empty.
    */
   public static AccountCollection list(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return requestCollection(classUrl(Account.class), params, AccountCollection.class, options);
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/accounts");
+    return requestCollection(url, params, AccountCollection.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="persons">
   /**
-   * List an account's persons.
+   * With <a href="/docs/connect">Connect</a>, you can create Stripe accounts for your users. To do
+   * this, you’ll first need to <a
+   * href="https://dashboard.stripe.com/account/applications/settings">register your platform</a>.
+   *
+   * <p>For Standard accounts, parameters other than <code>country</code>, <code>email</code>, and
+   * <code>type</code> are used to prefill the account application that we ask the account holder to
+   * complete.
    */
-  public PersonCollection persons()
-      throws StripeException {
-    return persons(null, null);
+  public static Account create(Map<String, Object> params) throws StripeException {
+    return create(params, (RequestOptions) null);
   }
 
   /**
-   * List an account's persons.
+   * With <a href="/docs/connect">Connect</a>, you can create Stripe accounts for your users. To do
+   * this, you’ll first need to <a
+   * href="https://dashboard.stripe.com/account/applications/settings">register your platform</a>.
+   *
+   * <p>For Standard accounts, parameters other than <code>country</code>, <code>email</code>, and
+   * <code>type</code> are used to prefill the account application that we ask the account holder to
+   * complete.
    */
-  public PersonCollection persons(Map<String, Object> params)
+  public static Account create(Map<String, Object> params, RequestOptions options)
```

Comment:
1) Create (standard): Static method to create a resource

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Account.java#L231-L235